src/bootstrap/src/ferrocene/secret_sauce.rs

```diff
@@ -0,0 +1,61 @@
+use std::path::PathBuf;
+
+use crate::TargetSelection;
+use crate::core::builder::{Builder, ShouldRun, Step};
+use crate::core::config::FerroceneSecretSauce;
+
+static DOWNLOAD_PREFIX: &str = "s3://ferrocene-ci-mirrors/coretest-secret-sauce/20250130";
```

Comment:
Nit: It's slightly confusing that the date is included in the download prefix. If I were to update the file, I would expect to change `COMMIT`, not `DOWNLOAD_PREFIX`. I'd move the date to `COMMIT` (or worst case a separate constant).